### DISCUSSION TITLE
Usage with Turborepo + Yarn Workspaces

### DISCUSSION THREAD

**Discussion by Freire71** [callstack/react-native-builder-bob#355] [Q&A]
Hi team, I'm using Yarn Workspaces with Turborepo. I've created an app under /apps and a component library under /packages
When I import the exported components in my app  they can be used without any problems (I've used bob to build my components library), however when I try to run Jest in a file that imports a component from the component library I get the following error:  
```
Invariant Violation: __fbBatchedBridgeConfig is not set, cannot invoke native modules

at invariant (../../packages/shared-components/node_modules/invariant/invariant.js:40:15)
at Object.<anonymous> (../../packages/shared-components/node_modules/react-native/Libraries/BatchedBridge/NativeModules.js:181:3)
at Object.<anonymous> (../../packages/shared-components/node_modules/react-native/Libraries/TurboModule/TurboModuleRegistry.js:11:23)
 ```

The only component that I'm exporting right now is a test button, I'm not creating any custom native component 
```
import * as React from 'react';
import {
  TouchableOpacity,
  StyleSheet,
  GestureResponderEvent,
  Text,
} from 'react-native';

export interface ButtonProps {
  text: string;
  onClick?: (event: GestureResponderEvent) => void;
}

export function Button({text, onClick}: ButtonProps) {
  return (
    <TouchableOpacity style={styles.button} onPress={onClick}>
      <Text style={styles.text}>{text}</Text>
    </TouchableOpacity>
  );
}

const styles = StyleSheet.create({
  button: {
    maxWidth: 200,
    textAlign: 'center',
    borderRadius: 10,
    paddingTop: 14,
    paddingBottom: 14,
    paddingLeft: 30,
    paddingRight: 30,
    fontSize: '15px',
    backgroundColor: '#2f80ed',
  },
  text: {
    color: 'white',
  },
});
```

**Comment by amerllica** [comment]
Dear @Freire71 

You said you've made a monorepo by using `TurboRepo` and Yarn Workspaces. and then you said you've got a `React Native` project in the `app` folder. (I wish it wasn't an expo app) Then you've got a package by using `builder-bob` in the `packages` folder.

I've got a problem integrating this really. My project in the `app` folder is a bare `React Native` app. Also, exactly the same as your configs my library was made by using `builder-bob`, But when I run `yarn install` in the root of monorepo, it tries to install my library from `npm` 😞 and doesn't link it.

I expected the `app/project` to get my library's codes from hoisted `node_modules` in the root of monorepo and then the top level `node_modules` link my library folder to the `packages/my-library`. In this way, by any change of library, I can have new changes directly in the `app/project`.

How you did do this configuration? Can I have any access to a reproduction of this monorepo? I need only a bare simple example.
